notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb

```diff
@@ -0,0 +1,1092 @@
+{
```

Comment:
<p><code>sngp_ood_labels</code> is not used later.  This should be renamed to <code>ood_labels</code> or other <code>ood_labels</code> to <code>sngp_ood_labels</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>Done!</p>